UnleashClient/features/Feature.py

```diff
@@ -58,6 +61,15 @@ def increment_stats(self, result: bool) -> None:
         else:
             self.no_count += 1
 
+    def count_variant(self, variant_name: str) -> None:
```

Comment:
Can I suggest this rather? Python doesn't really have a real way of making things private, the dunderscore does _things_ which make it a lot harder to access but not impossible, just a nuisance. The single underscore is enough to signal to developers that they probably shouldn't touch it unless they know what they're doing
```suggestion
    def _count_variant(self, variant_name: str) -> None:
```

Reply:
Yeah, that's cool! I'll take care of it! 🙋🏼

Reply:
Addressed in 9f1ec633a13cf3ce698a9658897f0b9d5e6cc124